lib/assessment/rubric_tester.py

```diff
@@ -302,57 +306,61 @@ def main():
                 print(f"Could not download dataset {options.dataset_name} lesson {lesson}")
                 logging.error(e)
 
-        # download experiment files
-        if not os.path.exists(experiment_lesson_prefix) or options.download:
-            check_aws_access()
+        # Clone parameter files from github if the repo does not already exist
+        if not os.path.exists(params_directory):
             try:
-                s3 = boto3.resource("s3")
-                get_s3_folder(s3, experiment_lesson_prefix)
+                get_params_github()
             except Exception as e:
-                print(f"Could not download experiment {options.experiment_name} lesson {lesson}")
+                print(f"Could not clone aitt_release_data repository. Please clone manually with git@github.com:code-dot-org/aitt_release_data.git")
                 logging.error(e)
```

Comment:
please reraise here, otherwise I get a longer error output and it's hard to know what part to look at.